quickstarts/bagisto/config.yml

```diff
@@ -0,0 +1,49 @@
+id: 9d758ec4-a261-06ef-be55-2e1f9cc55f2e
```

Comment:
Hey @suraj-webkul !

Since this is a new quickstart, we actually generate the ID for it on our end. No need to set up an existing ID 😄 

```suggestion

```

Reply:
Thanks @josephgregoryii